### PR TITLE
Add aws_sns_topic parameter to snowflake_pipe resource

### PR DESCRIPTION
**Pull request by velppa**
Add aws_sns_topic parameter to snowflake_pipe resource

### CODE REVIEW

**Review by inigofh — CHANGES_REQUESTED** [review]
Looks good to me but I'd suggest adding tests for this new feature. Although the tests are breaking by default in Travis, if we want to create a PR in the original repo this feature should be tested.

File: pkg/resources/stage_test.go [hunk 1]
```diff
@@ -55,5 +55,5 @@ func expectReadStageShow(mock sqlmock.Sqlmock) {
 	rows := sqlmock.NewRows([]string{
 		"created_on", "name", "database_name", "schema_name", "url", "has_credentials", "has_encryption_key", "owner", "comment", "region", "type", "cloud", "notification_channel", "storage_integration"},
 	).AddRow("2019-12-23 17:20:50.088 +0000", "test_stage", "test_db", "test_schema", "s3://load/test/", "N", "Y", "test", "great comment", "us-east-1", "EXTERNAL", "AWS", "NULL", "NULL")
-	mock.ExpectQuery(`^SHOW STAGES LIKE 'test_stage' IN DATABASE "test_db"$`).WillReturnRows(rows)
+	mock.ExpectQuery(`^SHOW STAGES LIKE 'test_stage' IN SCHEMA "test_db"."test_schema"$`).WillReturnRows(rows)
```
Comment (velppa) [inline pkg/resources/stage_test.go:58]: @inigofh this one was missed in #3.

Comment (inigofh) [reply pkg/resources/stage_test.go:58]: Well spotted! Thanks man.

**Review by inigofh — APPROVED** [review]
Looks good!

**Pull request merged**